src/metabase/search/config.clj

```diff
@@ -47,6 +47,11 @@
   [model-name]
   (Class/forName (format "metabase.models.%s.%sInstance" model-name (str/capitalize model-name))))
 
+(defn model-name->instance
+  "Given a model name as a string, return the specific instance"
+  [model-name]
+  (first (filter (fn [x] (= (str/capitalize model-name) (:name x))) searchable-models)))
```

Comment:
@camsaul is there a better way of doing string->model instance conversion?

Reply:
there's an explicit thingy in collections.clj but reusing it would be some weird-ass action-at-a-distance thing and I didn't want to whack it into a utils when I could whack it into _this_ utils

Reply:
(I guess whatever argument against taking from a map is basically the argument against taking from the map in the `model-name->class` that was written 3 months ago)